tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -0,0 +1,191 @@
+import { VALIDATION_ERROR } from 'const';
+import { CONTENT_SCRIPT_NOT_FOUND, CONTENT_SCRIPT_EMPTY } from 'messages/javascript';
+import JavaScriptScanner from 'scanners/javascript';
+
+describe('content_scripts_file_absent', () => {
+  it('should show an error when content script is missing', () => {
+    const code = `browser.tabs.executeScript({file: '/content_scripts/absentFile.js'});`;
+    const fileRequiresContentScript = 'file-requires-content-script.js';
+    const jsScanner = new JavaScriptScanner(code, fileRequiresContentScript, {
+      existingFiles: {
+        fileRequiresContentScript,
+      },
+      addonMetadata: {
+        id: 'test',
+      },
+    });
+
+    return jsScanner.scan()
+      .then(({ linterMessages }) => {
+        linterMessages.sort();
```

Comment:
do we need to sort the array of the linter messages here?
(given that it seems that we expect an array on a single element based on the asserion at line 21).